regen-network/regen-web
# fix: A user should be able to add/update a project created outside the app

## Description

Closes: #2039



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

On https://deploy-preview-2060--regen-marketplace.netlify.app/ecocredits/projects, all the projects that you are an admin of should appear (before the list was based on offchain projects).

For devs:
- create a new project on chain from the CLI (with ou without metadata, both are supported now too)
- try to edit this project from the UI

For @erikalogie 
You can try to edit this project which doesn't have any corresponding off-chain project yet nor any anchored metadata available on staging: https://deploy-preview-2060--regen-marketplace.netlify.app/project-pages/C01-004/edit
But be aware that as soon as you edit some info, it will create the associated off-chain project and/or update the on-chain metadata.
Verify the edited info shows up in the project edit forms initial values and on the project page.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
